src/modules/shared/types/media.ts

```diff
@@ -36,33 +36,3 @@ export interface MediaSearchFilter {
 }
 
 export type MediaSearchFilters = MediaSearchFilter[];
-
-export interface MediaInfo {
```

Comment:
Deze leeft best in shared, aangezien dit type in meerdere modules wordt herbruikt.